dataprofiler/reports/graphs.py

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env python3
+"""Contains functions for generating graph data report."""
+# !/usr/bin/env python3
```

Comment:
we need this? `!/user/bin/env python3`?

Reply:
It was already there.

Reply:
fine to leave imo

Reply:
for reference, https://predictivehacks.com/?all-tips=a-shebang-line-usr-bin-python3#:~:text=Its%20purpose%20is%20to%20define,script%20as%20python3%20file.py%20